### PR TITLE
fix: add variable.scss file path of cms and remove account and profile MFE

### PR DESCRIPTION
**Pull request by hinakhadim**
This PR fixes the `Undefined variable: "$color-hide-from-toc"` and also removes the header, the footer of profile and account MFE

### CODE REVIEW

File: changelog.d/20240611_174413_hina.khadim_fix_nightly.md [hunk 1]
```diff
@@ -0,0 +1 @@
+- [BugFix] Add variable.scss file path of edx-platform for CMS and remove header and  footer of profile and account MFE   (by @hinakhadim)
```
Comment (DawoudSheraz) [inline changelog.d/20240611_174413_hina.khadim_fix_nightly.md:1]: please update the changelog as we are no longer removing the header/footer in the PR.

Comment (hinakhadim) [reply changelog.d/20240611_174413_hina.khadim_fix_nightly.md:1]: Current nightly branch has header and footer. We have to remove them in nightly

Comment (DawoudSheraz) [reply changelog.d/20240611_174413_hina.khadim_fix_nightly.md:1]: Ah, got it. I was remembering it when they were removed previously, I did not see they have been added back.

File: tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables-v1.scss [hunk 2]
```diff
@@ -34,259 +28,3 @@ $uxpl-light-blue-base: rgb(242, 248, 251) !default;
 $uxpl-green-base: rgb(21, 55, 109) !default; // wcag2a compliant
 $uxpl-green-hover-active: lighten($uxpl-green-base, 8%) !default; // wcag2a compliant
 $uxpl-green-dark-hover-active:  lighten($uxpl-green-base, 8%) !default;
-
-$uxpl-pink-base: rgb(194, 56, 125) !default; // wcag2a compliant
-$uxpl-pink-hover-active: lighten($uxpl-pink-base, 7%) !default; // wcag2a compliant
-
-$uxpl-grayscale-x-back: rgb(245, 245, 245) !default; // UXPL grayscale, x-back
-
```
Comment (DawoudSheraz) [inline tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables-v1.scss:42]: I assume all of these variables are now coming from cms/static/sass/partials/cms/theme/variables-v1?

Comment (DawoudSheraz) [reply tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables-v1.scss:42]: nvm, got my answer from https://github.com/overhangio/tutor-indigo/blob/nightly/tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables-v1.scss

**Review by DawoudSheraz — APPROVED** [review]

File: tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables-v1.scss (hunk 2 above)
Comment (hinakhadim) [reply tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables-v1.scss:42]: Yes,  add import `cms/static/sass/partials/cms/theme/variables-v1` at the top. Only those variables exists that we've overrided.

File: changelog.d/20240611_174413_hina.khadim_fix_nightly.md (hunk 1 above)
Comment (hinakhadim) [reply changelog.d/20240611_174413_hina.khadim_fix_nightly.md:1]: Because `master` is merged into `nightly`.

**Pull request merged**